### ISSUE TITLE
Add Swift Package manager support

### ISSUE THREAD

**Issue by SmartAkashNara** [TTTAttributedLabel/TTTAttributedLabel#808]
Add Swift Package manager support

![Screenshot 2020-03-04 at 11 55 48 AM](https://user-images.githubusercontent.com/61772249/75851063-bb1cc780-5e0e-11ea-8a28-2f0d4b921735.png)

**Comment by vapor-pawelw** [comment]
Yes please! I love TTTAttributedLabel and it helps me in many projects, but it is one of few libs that don't let me switch to SPM completely

**Comment by sugitatestblue** [comment]
No supported yet?
it is 2022.

**Comment by sugitatestblue** [comment]
@TTTAttributedLabel
